src/io/iter_csv.cc

```diff
@@ -130,29 +130,74 @@ class CSVIter: public IIterator<DataInst> {
 DMLC_REGISTER_PARAMETER(CSVIterParam);
 
 MXNET_REGISTER_IO_ITER(CSVIter)
-.describe(R"code(Iterating on CSV files
+.describe(R"code(Returns the CSV file iterator.
 
-Assume there is CSV file at ``data/data.csv`` with content::
+In this function, `data_shape` parameter is used to set the shape of each line of the input data.
+If a row in an input file is `1,2,3,4,5,6` and `data_shape` is (3,2), each row
+in csv file will be reshaped to the data [[1,2],[3,4],[5,6]] of shape (3,2).
 
+By default, the CSVIter has `round_batch` parameter set to True. So, if `batch_size`
+is 3 and there are 4 total rows in CSV file, 2 more examples
+are consumed at the first round. If `reset()` function is called after first round,
```

Comment:
`reset` is a function, `reset()` is a function call. Which do you mean here? - I think the former but please check.